### ISSUE TITLE
[BUG] document maxsize for get-file response action

### ISSUE THREAD

**Issue by 111andre111** [elastic/security-docs#5130]
### Documentation links

https://www.elastic.co/guide/en/security/8.13/response-actions.html#_get_file

### Description

At this time we only document the [upload](https://www.elastic.co/guide/en/security/8.13/response-actions.html#_upload) response action having a limit of 100MB. 

We should do the same for `get-file` response action as that one has as well 100MB - hard coded and can't be changed:
https://github.com/elastic/kibana/blob/1be60ebd4c9d726581da2ce8131b6d41fa3c52db/x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/endpoint_action_response_codes.ts#L38-L42
It might be probably tied as well to the limit in Elasticsearch and we might link to it for description:
https://www.elastic.co/guide/en/elasticsearch/reference/8.13/general-recommendations.html#maximum-document-size

As of the Upload action I didn't find as well a documentation that with setting
`xpack.securitySolution.maxUploadResponseActionFileBytes`
the 100MB limit can be increased and what it means if we go over 100MB. So that could be added at the same time too.

### Which documentation set(s) does this bug apply to?

ESS and serverless

### Release version

Since get-file has been introduced in 8.7.

### Testing environment

Doesn't really need one. It's hard coded.

**Comment by intxgo** [comment]
> As of the Upload action I didn't find as well a documentation that with setting
> `xpack.securitySolution.maxUploadResponseActionFileBytes`
> the 100MB limit can be increased and what it means if we go over 100MB. So that could be added at the same time too.

I can see it clearly documented under the link you provided in description
<img width="860" alt="image" src="https://github.com/elastic/security-docs/assets/39905449/7900f4a3-4cb7-45a3-9338-de7bf0ebd834">

**Comment by 111andre111** [comment]
Yes, but that's the `upload` response action you are seeing @intxgo 
I am aiming for the `get-upload` response action.

**Comment by intxgo** [comment]
> `upload` = from Kibana to Endpoint -> can be configured in Kibana via advanced setting `fileUpload:maxFileSize` or in `kibana.yml` with setting `xpack.securitySolution.maxUploadResponseActionFileBytes` it seems with a max of 1GB...

The documentation screenshot I posted above says max is 100MB 😉 Do you mean you have tried to set it to 1GB and it worked?

**Comment by 111andre111** [comment]
@intxgo 
Seems we are misaligned a bit.
We are documenting a size limitation for the `upload` response action where you shared the screenshot from:
https://www.elastic.co/guide/en/security/8.13/response-actions.html#_upload
which has the direction `kibana -> endpoint`.

However we don't document a size limiation for `get-file` response action:
https://www.elastic.co/guide/en/security/8.13/response-actions.html#_get_file
which has the direction `endpoint -> kibana`. -> This is the one we are talking about this one in this issue here

I hope that makes it more clear.

**Comment by intxgo** [comment]
Agree, to clarify the kibana flag `xpack.securitySolution.maxUploadResponseActionFileBytes` applies only to `kibana -> endpoint` direction and is documented. I don't know what happens if you try to set it to a value larger than 100MB ignoring the documentation.

There's no Kibana flag to alter the max file size for the other direction. It's enforced by Endpoint.
